projects/step-core/styles/_core-variables.scss

```diff
@@ -8,7 +8,9 @@ $light-gray: #a0a0a0;
 $light-white-gray: #ddd;
 $light-blue-gray: #f2f4f7;
 $dark-blue: #0082cb;
-$darkest-blue: #293e50;
+$warn-yellow-background: #fff2e2;
+$warn-yellow-color: #ab793b;
+$warn-yellow-border: #ead5bd;
```

Comment:
I would prefer to have colours defined without mentioning the purpose. As decided in Trello it should be something like: 
$yellow-500
$yellow-200
$yellow-50

https://trello.com/c/bpjt3FEU/152-code-rule-colors-should-be-named-with-a-brightness-value-50-to-800-instead-of-unclear-names-like-lightblue-mediumblue-etc